hs/src/Reach/VerifySMT.hs

```diff
@@ -1,29 +1,31 @@
 module Reach.VerifySMT where
+
 -- XXX clean up function names and params to not refer to z3
 
-import Data.Maybe (catMaybes)
-import qualified Data.Set as S
-import qualified Data.Map as M
-import Data.Char (isDigit)
-import Text.Read (readMaybe)
+--- Maybe use Language.SMTLib2 in future
```

Comment:
It doesn't let you comment on import statements; it moved this to the top of the import block

Reply:
Maybe if the comment were on the same line?